src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -192,26 +192,17 @@ private static String getDwpIssuingOffice(SyaCaseWrapper syaCaseWrapper) {
         DwpAddressLookupService dwpLookup = new DwpAddressLookupService();
         String benefitType = syaCaseWrapper.getBenefitType().getCode();
         String result = null;
-        switch (Benefit.getBenefitByCode(benefitType)) {
-            case UC:
-            case CARERS_ALLOWANCE:
-            case BEREAVEMENT_BENEFIT:
-            case MATERNITY_ALLOWANCE:
-            case BEREAVEMENT_SUPPORT_PAYMENT_SCHEME:
-                result = dwpLookup.getDefaultDwpMappingByBenefitType(benefitType)
-                        .map(office -> office.getMapping().getCcd())
-                        .orElse(null);
-                break;
-            default:
-                if (!mrnIsNotProvided(syaCaseWrapper)) {
-                    String dwpIssuingOffice = syaCaseWrapper.getMrn().getDwpIssuingOffice();
-                    if (dwpIssuingOffice != null) {
-                        result = dwpLookup.getDwpMappingByOffice(benefitType, dwpIssuingOffice)
-                                .map(office -> office.getMapping().getCcd())
-                                .orElse(null);
-                    }
-                }
+
+        if (!mrnIsNotProvided(syaCaseWrapper) && StringUtils.isNoneBlank(syaCaseWrapper.getMrn().getDwpIssuingOffice())) {
```

Comment:
`!mrnIsNotProvided(syaCaseWrapper)`  This is a double negative, try to avoid it.
Instead of `StringUtils.isNoneBlank` use `isNotBlank` and use static imports where possible